code/modules/surgery/organs/stomach/stomach_ethereal.dm

```diff
@@ -6,11 +6,35 @@
 	var/crystal_charge = ETHEREAL_CHARGE_FULL
 	///used to keep ethereals from spam draining power sources
 	var/drain_time = 0
+	var/damagemodifer = 0.5
+	decay_factor = STANDARD_ORGAN_DECAY * 0.9 // fails around 16.5 minutes, the battery are one of the last organs to die (of the ones we have)
+	low_threshold_passed = "<span class='info'>You can feel your power draining as a feeling of pins and needles fills your abdomen.</span>"
+	high_threshold_passed = "<span class='warning'>Your abdomen is filled with the feeling of shocks and electric discharges!</span>"
+	high_threshold_cleared = "<span class='info'>The electric feeling in your stomach seems to fade away, leaving behind a feeling of pins and needles.</span>"
+	low_threshold_cleared = "<span class='info'>The pins and needles feeling seems to die out as your charge starts to feel far more stable.</span>"
 
 /obj/item/organ/stomach/ethereal/on_life(delta_time, times_fired)
 	. = ..()
 	adjust_charge(-ETHEREAL_CHARGE_FACTOR * delta_time)
+	if(damage > maxHealth)//cap battery damage
+		damage = maxHealth
+	if(damage < low_threshold)
+		handle_charge(owner, delta_time, times_fired)
+		return
+	adjust_charge(-0.8 * (damage / 5) * delta_time) //at 100 damage ethereals will loose 16 charge per second
```

Comment:
Please base this equation (and all like it) off of ETHEREAL_CHARGE_FACTOR so that it's updated accordingly should values be changed down the road.